warehouse/templates/search/results.html

```diff
@@ -84,6 +84,32 @@ <h3 class="package-snippet__title">
 {% endif %}
 {%- endmacro %}
 
+
+
+{% macro generate_markup(top_level, classifiers) -%}
+    {% set _id = top_level %}
+    {%- for classifier in classifiers %}
+     {% set children = classifiers[classifier] %}
+     {% if kwargs['parent'] != None %}
+      {% set _id = _id +' :: '+ kwargs['parent'] %}
+     {% endif %}
+    {% set id = _id + ' :: ' + classifier %}
+    <div class="checkbox-tree__item">
+      <li>
+        <input name="c" type="checkbox" id="{{ id }}" class="-js-form-submit-trigger checkbox-tree__checkbox" value="{{ id }}" {{ 'checked' if id in applied_filters else '' }}>
```

Comment:
is `applied_filters` available in the scope of this macro? you may need to pass it as an extra argument

Reply:
@yeraydiazdiaz `applied_filters` is available in the scope `generate_markup` macro. We wouldn't need to pass it as an argument.

Reply:
I understand, but I think it might be more explicit to take it as an argument. For example if we were to move the macro to a different file a developer reading the code would wonder what the value for `applied_filters` would be.